bindings/matrix-sdk-ffi/src/timeline.rs

```diff
@@ -614,3 +625,169 @@ pub impl MediaSourceExt for MediaSource {
         }
     }
 }
+
+#[derive(Clone, uniffi::Enum)]
+pub enum MembershipChange {
+    None,
```

Comment:
Why do we have this variant? If nothing changed, should we really add a timeline item?

Reply:
Why would we arbitrarily remove an item from the timeline? Isn't it the client's job to ignore events they don't want to display?

Besides the item might contain data that changed that we don't know about (eg an extra profile field) in the raw JSON, but the client does.